app/views/accounts/users_index.html.erb

```diff
@@ -0,0 +1,7 @@
+<ul>
+  <% @account.each do |account| %>
+    <li>
```

Comment:
As noted above, you don't want to be iterating through accounts in the users_index. Actually though you are doing the right thing here, it just looks like you are doing the wrong thing because of the way you named your variables